pool/pool.go

```diff
@@ -223,12 +228,57 @@ func (p *Pool) StoreTx(ctx context.Context, tx types.Transaction, ip string, isW
 		}
 	}
 
+	gasPrices, err := p.GetGasPrices(ctx)
```

Comment:
This makes a query to the database to get the gasPrices, this could have some impact in the performance, as we are quering for each txs we need to validate. We have the pool.GetL1AndL2GasPrice function that gets this info from a shared struct that is updated each 5 seconds (IntervalToRefreshGasPrices), it is update in the refreshGasPrices function (that is called from a go function). The pool.GetL1AndL2GasPrice is the function used in the sequencer to get the proces used during the egp calculation